docs/org.eclipse.n4js.spec/chapters/05_classifiers/classifiers-part-02-members.adoc

```diff
@@ -786,6 +786,61 @@ new C( { s: "Hello", weird: true } );
 
 ====
 
+Restriction when initializing interface fields via @Spec constructor [[restriction-interface-field-spec-style-constructor]] ::
+
+In most cases, interface definitions in `n4jsd` files simply declare functions and fields that are supposed to by provided by the runtime environment.
+As a result, there are restrictions as to whether fields of interface defined in `n4jsd` files can initialized via `@Spec` constructors or not.
+In particular, fields of an interface declared in a `n4jsd` file  cannot be initialized via @Spec constructor if the interface
+
+. is a built-in or
+. does not have an `@N4JS` annotation
+
+The following example illustrates this restriction.
+
+.Interface fields that cannot be initialized via @Spec constructors
+[example]
+====
+
+.Inf.n4jsd
+[source,n4js]
+----
+export external interface I  {
+    public m: string;
+}
+
+@N4JS
+export external interface J  {
+    public n: string;
+}
```

Comment:
you are not using interface `J` in the second file.
Either
* remove interface `J`, OR
* add a class `D` implementing `J` and correpsonding lines to show on the console that initialization actually works for field n in J.